tests/factories/recurring_reservation.py

```diff
@@ -69,7 +69,7 @@ def create_with_matching_reservations(cls, **kwargs: Any) -> RecurringReservatio
         space = SpaceFactory.create(unit=series.reservation_unit.unit)
         series.reservation_unit.spaces.add(space)
 
-        weekdays: list[int] = [int(val) for val in series.weekdays.split(",") if val != ""]
+        weekdays: list[int] = [int(val) for val in series.weekdays.split(",") if val]
```

Comment:
This is wrong, it skips Mondays (value=0) since 0 is falsy.

Reply:
`0` is falsy, but `"0"` is truthy.
Since `series.weekdays` is always a string (we can't split an `int` anyway) this shouldn't affect the logic at all.

Reply:
Ah, that's right, my mistake.